src/simsopt/util/adjust_magnet_angles.py

```diff
@@ -0,0 +1,699 @@
+"""
+    This file contains a class for reading in FOCUS-style data files. 
+
+    This file was copied over from the MAGPIE code used for generating 
+    permanent magnet structures. All credit for this code is thanks 
+    to the PM4Stell team and Ken Hammond for his consent to use this file
+    and work with the permanent magnet branch of SIMSOPT.
+"""
+
+import numpy as np
+import pandas as pd
```

Comment:
This line introduces a dependency on pandas. Every new dependency is an opportunity for installation to fail, so we should look carefully at this. Also, some of the unit tests are failing due to this import. It appears that pandas is only used in the function `moments_file()`, which is never called. Can we delete that function and so remove this import of pandas?